docs/source/contributing/docs.md

```diff
@@ -0,0 +1,69 @@
+(contributing-docs)=
+
+# Contributing Documentation
+
+Documentation is often more important than code. This page helps you get set up on how to contribute to Pytest JupyterHub's documentation.
+
+## Building documentation locally
+
+We use [sphinx](https://www.sphinx-doc.org) to build our documentation. It takes our documentation source files (written in [markdown](https://daringfireball.net/projects/markdown/) or [reStructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) & stored under the `docs/source` directory) and converts it into various
+formats for people to read. To make sure the documentation you write or change renders correctly, it is good practice to test it locally.
+
+1. Make sure you have successfully completed {ref}`contributing/setup`.
+
+2. Install the packages required to build the docs.
+
+   ```bash
+   python3 -m pip install -r docs/requirements.txt
+   ```
+
+3. Build the html version of the docs. This is the most commonly used
+   output format, so verifying it renders correctly is usually good
+   enough.
+
+   ```bash
+   cd docs
+   make html
+   ```
+
+   This step will display any syntax or formatting errors in the documentation, along with the filename / line number in which they occurred. Fix them, and re-run the `make html` command to re-render the documentation.
+
+4. View the rendered documentation by opening `_build/html/index.html` in
+   a web browser.
+
+   :::{tip}
```

Comment:
> I believe this a leftover from https://github.com/jupyterhub/pytest-jupyterhub/commit/8133acc64ee715c260739d3b1267bac072bd099a and it's not actually used.

No, I still use it here. It just did not seem to work with an `example` directive I wanted to add which I decided was unimportant anyway. I use it in the setup docs as well.